qrhl/qrhl.el

```diff
@@ -0,0 +1,115 @@
+;; Author: Dominique Unruh
+
+(require 'proof)
+
+(load-library "qrhl-input")
+
+(defun qrhl-find-and-forget (span)
+  (proof-generic-count-undos span))
+  
+(defvar qrhl-home (file-name-directory (directory-file-name (file-name-directory (directory-file-name (file-name-directory load-file-name))))))
```

Comment:
What does this?

Reply:
Well spotted. This is some code for determining where the binary is, but it does not make sense in a general context. (I.e., when the PG-fork is not integrated into the qrhl-tool distribution itself.) I removed it and replaced it by a proper customizable binary path (543aa77).